.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
     - COMPOSER_ARGS="--no-interaction"
     - COVERAGE_DEPS="php-coveralls/php-coveralls"
     - TESTS_ZEND_HTTP_CLIENT_ONLINE=true
+    - PHPSTAN_DEPS="phpstan/phpstan:^0.10.3"
```

Comment:
In other repositories we are adding it to composer dev requirements. Maybe we should do the same here?
Then it will be much easier to run it locally. See: https://github.com/zendframework/zend-expressive/blob/master/composer.json#L40

We can also update to version 0.11 :)

Reply:
Someone else told me to remove it and to add it in CI.